docs/ReleaseNotes.md

```diff
@@ -4,11 +4,13 @@ This document contains a log of changes to the FoundationDB Record Layer. It aim
 
 As the [versioning guide](Versioning.md) details, it cannot always be determined solely by looking at the version numbers whether one Record Layer version contains all changes included in another. In particular, bug fixes and backwards-compatible changes might be back-ported to or introduced as patches against older versions. To track when a patch version has been included in the main release train, some releases will say as a note that they contain all changes from a specific patch.
 
-## 3.1
+## 3.2
 
-### Breaking Changes
+### Breaking changes
```

Comment:
Is it intentional that this is different case from the following sections for older releases?

Reply:
No, sorry, that was a typo. Fixed